### PR TITLE
Fix parsing errors for `mc ilm set` command

### PR DESCRIPTION
**Pull request by poornas**
fix on top of PR https://github.com/minio/mc/pull/3348

### CODE REVIEW

**Review by nitisht — APPROVED** [review]
LGTM

**Comment by nitisht** [comment]
Thanks for the help with this @poornas . I sent a fix with small updates to example text, other than that LGTM

File: cmd/ilm-add.go [hunk 1]
```diff
@@ -44,43 +44,30 @@ FLAGS:
...
 
 var ilmAddFlags = []cli.Flag{
-	cli.StringFlag{
-		Name:  "id",
-		Usage: "id for the rule, should be a unique value",
-	},
-	cli.StringFlag{
-		Name:  "prefix",
-		Usage: "prefix to apply the lifecycle configuration rule",
-	},
```
Comment (BenoitKnecht) [inline cmd/ilm-add.go:76]: Why get rid of the `prefix` option? It's no longer possible to create lifecycle policies that only match a subset of objects based on their prefix.

File: cmd/ilm-add.go [hunk 2]
```diff
@@ -44,43 +44,30 @@ FLAGS:
...
+  1. Add expiration rule on mybucket.
+     {{.Prompt}} {{.HelpName}} --expiry-days "200" myminio/mybucket
+
+  2. Add expiry and transition date rules on a prefix in mybucket.
+     {{.Prompt}} {{.HelpName}} -expiry-date "2025-09-17" --transition-date "2025-05-01" \
+          --storage-class "GLACIER" myminio/mybucket/doc
+
+  3. Add expiry and transition days rules on a prefix in mybucket.
+     {{.Prompt}} {{.HelpName}} -expiry-days "300" --transition-days "200" \
+          --storage-class "GLACIER" myminio/mybucket/doc
```
Comment (BenoitKnecht) [inline cmd/ilm-add.go:59]: And it makes examples 2 and 3 inaccurate here, the description mentions a prefix, but the policy that gets added doesn't match any particular prefix.

Comment (BenoitKnecht) [reply cmd/ilm-add.go:59]: Ooohh, never mind, I just noticed that the prefix is now simply added after the bucket name!

Sorry for the noise.

**Pull request merged**